tests/ops/fixtures/application_fixtures.py

```diff
@@ -592,6 +590,291 @@ def erasure_policy_two_rules(
         pass
 
 
+@pytest.fixture(scope="function")
+def erasure_policy_complete_mask(
+    db: Session,
+    oauth_client: ClientDetail,
+) -> Generator:
+    erasure_policy = Policy.create(
+        db=db,
+        data={
+            "name": "example erasure policy string rewrite",
+            "key": "example_erasure_policy_string_rewrite",
+            "client_id": oauth_client.id,
+        },
+    )
+
+    user__device__ip_address_rule = Rule.create(
```

Comment:
Can you change the double underscores `__` to single underscores for all these variable names?